sdk/servicebus/service-bus/test/browserified/receiveAndDeleteMode.spec.ts

```diff
@@ -0,0 +1,448 @@
+    await sendReceiveMsg(testMessages, autoCompleteFlag);
+
+    await testPeekMsgsLength(receiverClient, 0);
+  }
+
+  it("Partitioned Queue: With auto-complete enabled, no settlement of the message removes message", async function(): Promise<
+    void
+  > {
+    await beforeEachTest(TestClientType.PartitionedQueue, TestClientType.PartitionedQueue);
+    await testNoSettlement(true);
+  });
+
+  it("Partitioned Queue with Sessions: With auto-complete enabled, no settlement of the message removes message", async function(): Promise<
+    void
+  > {
+    await beforeEachTest(
+      TestClientType.PartitionedQueueWithSessions,
+      TestClientType.PartitionedQueueWithSessions,
+      true
+    );
+    await testNoSettlement(true, true);
+  });
+
+  it("Partitioned Queue: With auto-complete disabled, no settlement of the message removes message", async function(): Promise<
+    void
+  > {
+    await beforeEachTest(TestClientType.PartitionedQueue, TestClientType.PartitionedQueue);
+    await testNoSettlement(false);
+  });
+
+  it("Partitioned Queue with Sessions: With auto-complete disabled, no settlement of the message removes message", async function(): Promise<
+    void
+  > {
+    await beforeEachTest(
+      TestClientType.PartitionedQueueWithSessions,
+      TestClientType.PartitionedQueueWithSessions,
+      true
+    );
+    await testNoSettlement(false, true);
+  });
+
+  it("Partitioned Subscription with Sessions: With auto-complete disabled, no settlement of the message removes message", async function(): Promise<
+    void
+  > {
+    await beforeEachTest(
+      TestClientType.PartitionedTopicWithSessions,
+      TestClientType.PartitionedSubscriptionWithSessions,
+      true
+    );
+    await testNoSettlement(false, true);
+  });
+});
+
+describe("Unsupported features in ReceiveAndDelete mode", function(): void {
+  afterEach(async () => {
+    await afterEachTest();
+  });
+  async function sendReceiveMsg(testMessages: SendableMessageInfo): Promise<ServiceBusMessage> {
+    await sender.send(testMessages);
+    const msgs = await receiver.receiveMessages(1);
+
+    should.equal(Array.isArray(msgs), true, "`ReceivedMessages` is not an array");
+    should.equal(msgs.length, 1, "Unexpected number of messages");
+    should.equal(msgs[0].body, testMessages.body, "MessageBody is different than expected");
+    should.equal(msgs[0].messageId, testMessages.messageId, "MessageId is different than expected");
+    should.equal(msgs[0].deliveryCount, 0, "DeliveryCount is different than expected");
+
+    return msgs[0];
+  }
+
+  const testError = (err: Error, operation: DispositionType) => {
+    should.equal(
+      err.message,
+      getErrorMessageNotSupportedInReceiveAndDeleteMode(`${operation} the message`),
+      "ErrorMessage is different than expected"
+    );
+    errorWasThrown = true;
+  };
+
+  async function testSettlement(operation: DispositionType, useSessions?: boolean): Promise<void> {
```

Comment:
So we still want to keep the receiveAndDeleteMode tests? I thought we wanted to remove all of them

Reply:
Since the check that throws this error of unsupported feature is in the sdk and is only for ReceiveAndDelete mode, keeping these (unsupported feature) tests is the right thing to do.
I see that we now do this just for the sessions and that is fine.